crates/networking/src/gossip.rs

```diff
@@ -24,6 +24,9 @@ use tokio::sync::mpsc::UnboundedSender;
 use tokio::sync::{Mutex, RwLock};
 
 use crate::networking::{Network, ParticipantInfo, ProtocolMessage};
+use gadget_std as std;
```

Comment:
why is this needed? Prefer to keep it quite explicit.

Reply:
@tbraun96 You can use `gadget_macros::ext::std`

Reply:
That's one more dependency I have to add.